cmd/ketch/builder_list.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+)
+
+const builderListHelp = `
+List CNCF registered builders
+`
+
+type BuilderListEntry struct {
+	Vendor      string
+	Image       string
+	Description string
+}
+
+var builderList = []BuilderListEntry{
+	{
+		Vendor:      "Google",
+		Image:       "gcr.io/buildpacks/builder:v1",
+		Description: "GCP Builder for all runtimes",
+	},
+	{
+		Vendor:      "Heroku",
+		Image:       "heroku/buildpacks:18",
+		Description: "heroku-18 base image with buildpacks for Ruby, Java, Node.js, Python, Golang, & PHP",
+	},
+	{
+		Vendor:      "Heroku",
+		Image:       "heroku/buildpacks:20",
+		Description: "heroku-20 base image with buildpacks for Ruby, Java, Node.js, Python, Golang, & PHP",
+	},
+	{
+		Vendor:      "Paketo Buildpacks",
+		Image:       "paketobuildpacks/builder:base",
+		Description: "Small base image with buildpacks for Java, Node.js, Golang, & .NET Core",
+	},
+	{
+		Vendor:      "Paketo Buildpacks",
+		Image:       "paketobuildpacks/builder:full",
+		Description: "Larger base image with buildpacks for Java, Node.js, Golang, .NET Core, & PHP",
+	},
+	{
+		Vendor:      "Paketo Buildpacks",
+		Image:       "paketobuildpacks/builder:tiny",
+		Description: "Tiny base image (bionic build image, distroless run image) with buildpacks for Golang",
+	},
+}
+
+func newBuilderListCmd(cfg config, out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "List CNCF registered builders",
+		Long:  builderListHelp,
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			writeBuilders(cfg, out)
+		},
+	}
+	return cmd
+}
+
+func getUserBuilders(cfg config) []BuilderListEntry {
+	var userBuilders []BuilderListEntry
+	for _, builder := range cfg.GetKetchConfigObject().AdditionalBuilders {
+		entry := BuilderListEntry{
+			Vendor:      builder.Vendor,
+			Image:       builder.Image,
+			Description: builder.Description,
+		}
+		userBuilders = append(userBuilders, entry)
+	}
+
+	return userBuilders
+}
+
+func writeBuilders(cfg config, out io.Writer) {
```

Comment:
It would be possible to print them separately, but I feel that printing them together is fine. I think most users won't be making use of custom builders in the first place. I think the users that do choose to use a custom builder and add its details to the config.toml would be providing a builder that is reliable enough within their context for other users (in their group/organization) to use even if they don't realize that it is a custom builder.